homeassistant/components/vasttrafik/sensor.py

```diff
@@ -36,7 +41,7 @@
     {
         vol.Required(CONF_KEY): cv.string,
         vol.Required(CONF_SECRET): cv.string,
-        vol.Optional(CONF_DEPARTURES): [
+        vol.Required(CONF_DEPARTURES): [
```

Comment:
This behavior change seems a little out of place in a type hint change. Can this come in a follow up?

Reply:
Or this is already a problem eg.... `TypeError: 'NoneType' object is not iterable`

Reply:
The documentation already specifies it as required: https://www.home-assistant.io/integrations/vasttrafik/
So maybe it should considered a bugfix - but it seemed small enough to fold into this.

As I said in the description, the first pass was 100% pure type hints, but this second pass includes minor tweaks to each component.